bin/README.md

```diff
@@ -0,0 +1,75 @@
+# Mini Tree Server
+
+Mini Tree Server is an Axum-based server that encapsulates the functionality of the Mini Tree Generator.
+
+## Test Mini Tree Server
+
+First, to start the Mini Tree Server, use the command:
+
+```bash
+  cargo run --release --bin mini-tree-server
+```
+
+Alternatively, if you have the summa-aggregation-mini-tree image locally, can run the server with this command:
```

Comment:
Can you specify that these commands has to be run from root folder? Otherwise, the command `bash ./scripts/test_sending_entry.sh` will fail if you run it from `bin`

Reply:
Added comment [c8a2d7f](https://github.com/summa-dev/summa-aggregation/pull/3/commits/c8a2d7f0ae2411b5e44a86837bc43dda5256f6be)